test.properties

```diff
@@ -4224,37 +4224,6 @@
     test.batch.dist.app.servers[blade]=tomcat
     test.batch.names[blade]=blade-samples
 
-    #
-    # Blogs
-    #
-
-    test.batch.class.names.includes[blogs]=\
```

Comment:
@adolfopa, if we remove the suit, we should also remove it from ci.properties. Here: https://github.com/liferay/liferay-portal/blob/20c16b763e47400f6e4d7cc1f462a054278d1d4b/ci.properties#L46

Also when we add suits